.gitlab/benchmarks.yml

```diff
@@ -134,6 +136,8 @@ benchmark-serverless:
   script:
     - git clone https://gitlab-ci-token:${CI_JOB_TOKEN}@gitlab.ddbuild.io/DataDog/serverless-tools.git ./serverless-tools && cd ./serverless-tools
     - ./ci/check_trigger_status.sh
+  except:
+    - main  # fails on main - ideally it would succeed
```

Comment:
QQ - Do we know what's causing the failures? I just checked a few recent ones on main and they look to be cancelled?

Reply:
I don't